libethereum/BlockChainSync.cpp

```diff
@@ -819,18 +812,18 @@ void BlockChainSync::onPeerNewBlock(std::shared_ptr<EthereumPeer> _peer, RLP con
     case ImportResult::FutureTimeUnknown:
     case ImportResult::UnknownParent:
     {
-        _peer->m_unknownNewBlocks++;
-        if (_peer->m_unknownNewBlocks > c_maxPeerUknownNewBlocks)
+        m_host.incrementPeerUnknownNewBlocks(_peerID);
```

Comment:
This is quite different style than the next line where a filed of peer status struct is accessed directly.

Reply:
Good point, I think I'll make a class out of `EthereumPeerStatus`